.github/workflows/publish-artifact.yml

```diff
@@ -60,27 +60,35 @@ jobs:
           echo "PKG_LATEST_TAG=${DOCKER_BASE_NAME}:latest" >> $GITHUB_ENV
           echo "HUB_TAG=${DOCKER_HUB_BASE_NAME}:${TAG_NAME}" >> $GITHUB_ENV
           echo "HUB_LATEST_TAG=${DOCKER_HUB_BASE_NAME}:latest" >> $GITHUB_ENV
-      - name: Build image
-        run: |
-          docker build . -t "${PKG_TAG}" --build-arg SECRETLINT_VERSION=${SECRETLINT_VERSION}
-          docker tag "${PKG_TAG}" "${HUB_LATEST_TAG}"
-          docker tag "${PKG_TAG}" "${HUB_TAG}"
-        working-directory: ./publish/docker
-      - name: Print version
-        run: |
-          docker run --rm ${PKG_TAG} --version
-        working-directory: ./publish/docker
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
       - name: Login to Registries
         run: |
-          echo "${GITHUB_TOKEN}" | docker login docker.pkg.github.com -u azu --password-stdin
+          echo "${GITHUB_TOKEN}" | docker login ghcr.io -u azu --password-stdin
           echo "${DOCKER_HUB_TOKEN}" | docker login -u efcl --password-stdin
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           DOCKER_HUB_TOKEN: ${{ secrets.DOCKER_HUB_TOKEN }}
-      - name: Push to GitHub Packages
+      - name: Build image
+        run: |
+          docker buildx build . \
+            --build-arg SECRETLINT_VERSION=${SECRETLINT_VERSION} \
+            --load \
+            -t "${PKG_TAG}"
```

Comment:
There is a need to build for the local platform before build for the multi-platform. Because multi-platform builds do not allow for local image storage. The locally stored image is used in the `Print version` step.